resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -514,5 +548,133 @@ namespace {{packageName}}.Client
             return buffer;
         }
         {{/supportsUWP}}
+
+    public class RetryConfiguration
+    {
+    private long backoffIntervalMs = 300000L;
+    private long retryAfterDefaultMs = 3000L;
+    private int maxRetryTimeSec = 0;
+
+    public long BackOffIntervalMs
+    {
+        get
+        {
+            return backoffIntervalMs;
+        }
+
+        set
+        {
+            if (value < 0)
+            {
+                throw new ArgumentException("backoffInterval should be a positive integer");
```

Comment:
Use the public variable name in the error message (here and the rest below)

Reply:
fixed